### PR TITLE
delete branch

### PR DESCRIPTION
**Pull request by maxmodlol**
mobile-first /css rules

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -18,459 +18,559 @@
...
-          <p style="font-size: 14px;font-family: Arial;font-weight: 600;">`Shared resentments ' but Russia  ties could be 'awkward 'for China</p>
-          <p style="font-size: 12px ;font-family: Roboto;color: gray;object-fit:cover;overflow: hidden"> Russia and china's leaders have met more than 30 times since 2013 ,but the Ukraine crises will test the relationship.</p>
-        </div>
+    <div class="header_right">
+         <div class="header_right--big">
+             <img src="img/000_323T6QE.jpg"  >
 
+         <div class="header_text">
+        <h2 style="">`Shared resentments ' but Russia  ties could be 'awkward 'for China</h2>
+        <p style=""> Russia and china's leaders have met more than 30 times since 2013 ,but the Ukraine crises will test the relationship for the statments of the crises in this day .</p>
```
Comment (Abdallah-Via) [inline index.html]: remove style attribute

File: index.html [hunk 2]
```diff
@@ -18,459 +18,559 @@
...
-        </div>
-        <div style=";float: right;margin-left: 10px;margin-top: -15px;padding-top: 1px;height: 150px">
-          <p style="font-size: 14px;font-family: Arial;font-weight: 600;">`Shared resentments ' but Russia  ties could be 'awkward 'for China</p>
-          <p style="font-size: 12px ;font-family: Roboto;color: gray;object-fit:cover;overflow: hidden"> Russia and china's leaders have met more than 30 times since 2013 ,but the Ukraine crises will test the relationship.</p>
-        </div>
+    <div class="header_right">
+         <div class="header_right--big">
+             <img src="img/000_323T6QE.jpg"  >
 
+         <div class="header_text">
```
Comment (Abdallah-Via) [inline index.html]: header_title

File: index.html [hunk 3]
```diff
@@ -18,459 +18,559 @@
...
+    }
+    function myFunction() {
+      var x = document.getElementById("myLinks");
+      if (x.style.display === "block") {
+        x.style.display = "none";
+      } else {
+        x.style.display = "block";
+      }
+    }
+  </script>
```
Comment (Abdallah-Via) [inline index.html]: The script tag should be before </body>

File: index.html [hunk 4]
```diff
@@ -18,459 +18,559 @@
...
+</div>
+</nav>
+<div class="header">
+    <div class="header_content">
+         <div class="header_img--big">
+            <img src="img/tank.jpg" >
+         </div>
+         <div class="square" ></div>
+         <div class="header_text--main">
+             <p class="live--red">
```
Comment (Abdallah-Via) [inline index.html]: BEM?
if you want to name css attribute as color name, make it alone

File: index.html [hunk 5]
```diff
@@ -18,459 +18,559 @@
...
+    <a>
+      <img src="img/live-streaming.png" class="liveicon--small">
+    </a>
+  </div>
+  <div class="ham_menu">
+    <a href="javascript:void(0);" class="ham_menu_icon" onclick="myFunction()">
+    <i class="fa fa-bars"  onclick="myfun(this)"></i>
+  </a>
+  </div>
+  <div class="mobile-container">
```
Comment (Abdallah-Via) [inline index.html]: use semantic html

File: index.html [hunk 6]
```diff
@@ -18,459 +18,559 @@
...
+      <div  class="PopUp_nav" id="myLinks">
```
Comment (Abdallah-Via) [inline index.html]: use ul/li in list link (all file)

File: index.html [hunk 7]
```diff
@@ -18,459 +18,559 @@
...
+            <h3>Covid :DO we know the real death tool?</h3>
+        </div>
 
 
-</div>
+    </div>
 
 
 </div>
+<div class="news--large--tall">
```
Comment (Abdallah-Via) [inline index.html]: please check indentation

File: css/csshome.css [hunk 8]
```diff
@@ -1,383 +1,1203 @@
...
-  height:60px;
+  margin-left: 60px;
+  margin-top:60px ;
   box-shadow: 0 1px 2px 0 rgba(0,0,0,.2);
-  overflow: hidden;
-  position: fixed;
-  top: 0px;
-  background: white;
+}
+.header_img--big img {
```
Comment (Abdallah-Via) [inline css/csshome.css]: if the style for image, make the attribute on image, not parent div

File: index.html [hunk 9]
```diff
@@ -18,459 +18,559 @@
...
-          <p style="font-size: 14px;font-family: Arial;font-weight: 600;">`Shared resentments ' but Russia  ties could be 'awkward 'for China</p>
-          <p style="font-size: 12px ;font-family: Roboto;color: gray;object-fit:cover;overflow: hidden"> Russia and china's leaders have met more than 30 times since 2013 ,but the Ukraine crises will test the relationship.</p>
-        </div>
 
-      </div>
+</div>
+</nav>
+<div class="header">
+    <div class="header_content">
+         <div class="header_img--big">
```
Comment (Abdallah-Via) [inline index.html]: bem?

File: index.html [hunk 10]
```diff
@@ -1,476 +1,534 @@
...
+        </div>
+        <button class="accordion">Connect</button>
+        <div class="panel">
+            <li><a href="">Contact Us</a></li>
+            <li><a href="">Apps</a></li>
+            <li><a href="">Channel Finder</a></li>
+            <li><a href="">TV Shecdule</a></li>
+            <li><a href="">Podcast</a></li>
+            <li><a href="">Submit A Tip</a></li>
+        </div>
```
Comment (Abdallah-Via) [inline index.html]: you cannot wrapping li with div, Instead of wrapping li with div, use ul

File: index.html [hunk 11]
```diff
@@ -1,476 +1,534 @@
...
+            </h2>
+            <p class="news_text--defualt"> Russia and china's leaders have met more than 30 times since 2013 ,but the
+                Ukraine crises will test the relationship for the statments of the crises in this day .</p>
+        </div>
+    </div>
+    <div class="footer_accordion">
+        <button class="accordion">About</button>
+        <div class="panel">
+            <div>
+                <li><a href="">About Us</a></li>
```
Comment (Abdallah-Via) [inline index.html]: don't use empty ref, make it = "#"

File: index.html [hunk 12]
```diff
@@ -1,476 +1,534 @@
...
+                <p> Rejection in the diplomacey of bitcoin its a fake new .</p>
+            </div>
+        </div>
+    </section>
+    <div class="related_topic">
+        <div>
+            <p>MOST READ </p>
+            <div class="square"></div>
+        </div>
+        <div class="related_topic--text">
```
Comment (Abdallah-Via) [inline index.html]: this is not following BEM

File: index.html [hunk 13]
```diff
@@ -1,476 +1,534 @@
...
+            <p> Ukraine is now responaine on the russian attacks.</p>
+            <p> Ukraine is now responaine on the russian attacks.</p>
+            <p> Ukraine is now responaine on the russian attacks.</p>
+            <p> Ukraine is now responaine on the russian attacks.</p>
+            <p> Ukraine is now responaine on the russian attacks.</p>
+            <p> Ukraine is now responaine on the russian attacks.</p>
+            <p> Ukraine is now responaine on the russian attacks.</p>
+        </div>
+    </div>
+    <div class="news--small">
```
Comment (Abdallah-Via) [inline index.html:237]: not following BEM

File: index.html [hunk 14]
```diff
@@ -1,476 +1,534 @@
...
+                <li><a href="#about">Feature</a></li>
+                <li><a href="#about">Economy</a></li>
+                <li><a href="#about">Opinion</a></li>
+                <li><a href="#about">Video</a></li>
+                <li><a href="#about">More</a></li>
+            </nav>
+        </div>
+    </nav>
+    <div class="main_news">
+        <div class="news_content">
```
Comment (Abdallah-Via) [inline index.html]: not following BEM

File: index.html [hunk 15]
```diff
@@ -1,476 +1,534 @@
...
+                    LIVE UPDATES </p>
+                <h2> Ukrine to declare state of emergency as threat of war looms</h2>
+                <p class="text--gray"> After calling on citizines to leave Russia and imposing new sanctions ,Ukraine
+                    heads towards emergency rule</p>
+                <p class="text--bold"> Related Stories </p>
+                <p>Nord stream 2 ,What you should know about Russian's gas pipline to Europe Russian-Ukraine crises 5
+                    things to know about Putin's hold move what sections have been imposed on Russia?</p>
+            </div>
+        </div>
+        <div class="news_right">
```
Comment (Abdallah-Via) [inline index.html]: not following BEM

File: index.html (hunk 13 above)
Comment (maxmodlol) [reply index.html:237]: isn't  small  considered as a  modifier  in here ? or i should make it news separated and small as a separated class

**Review by Abdallah-Via — APPROVED** [review]

**Pull request merged**